Shared/Reusable/ListButton.swift

```diff
@@ -12,18 +12,29 @@ import SwiftUI
 struct ListButton: View {
     var image: Image
     var text: String
-    var action: () -> Void
+    var action: () -> Void = {}
     
     var body: some View {
         Button(action: action) {
             HStack(spacing: 10) {
-                #if os(iOS)
-                HStack { image.font(.system(size: 20)) }
-                    .frame(width: 30)
+                image.font(.system(size: 20))
+                
                 Text(text)
-                #endif
             }
         }
         .foregroundColor(.primary)
     }
 }
+
+#if DEBUG
+
+struct ListButton_Previews: PreviewProvider {
+    static var previews: some View {
+        ListButton(
+            image: Images.refresh,
+            text: "Refresh Data"
+        )
+    }
+}
+
+#endif
```

Comment:
Please revert these changes too

Reply:
I understood your previous points, but do you mind to explain why revert this preview here? Honestly, I don’t see any reason to not have a preview for some view when we are using SwiftUI. Would you mind share your points?

Reply:
Okay, I agree that we can keep the change but could you remove it here, I’ll merge this PR then you can create another PR with these changes? Just to keep things organized :)

Reply:
If you wish, here we go! ;)